src/goto-programs/xml_goto_trace.cpp

```diff
@@ -25,31 +25,32 @@ Author: Daniel Kroening
 #include "structured_trace_util.h"
 #include "xml_expr.h"
 
-bool full_lhs_value_includes_binary(
-  const goto_trace_stept &step,
-  const namespacet &ns)
-{
-  return can_cast_type<floatbv_typet>(step.full_lhs_value.type());
-}
-
 xmlt full_lhs_value(const goto_trace_stept &step, const namespacet &ns)
 {
-  xmlt full_lhs_value{"full_lhs_value"};
+  xmlt xml{"full_lhs_value"};
```

Comment:
:pick: Personally don't love this name either just cos the type tells you its xml /shrug

Reply:
Ok. I am going to make this `value_xml` instead.